main/src/main/java/org/objenesis/strategy/StdInstantiatorStrategy.java

```diff
@@ -62,9 +62,14 @@
             }
             return new AccessibleInstantiator<T>(type);
          }
-         // The UnsafeFactoryInstantiator would also work. But according to benchmarks, it is 2.5
-         // times slower. So I prefer to use this one
-         return new SunReflectionFactoryInstantiator<T>(type);
+         if (PlatformDescription.VM_VERSION.startsWith("9")) {
+            // UnsafeFactoryInstantiator doesn't work on JDK9 anymore
```

Comment:
Ouch, copy-paste error :( Of course comment should be:
// SunReflectionFactoryInstantiator doesn't work on JDK9 anymore